--- a/libobs/obs-module.c	2022-10-01 23:19:08.216586034 +0300
+++ b/libobs/obs-module.c	2022-10-01 23:45:51.944181412 +0300
@@ -298,13 +298,13 @@
 		return;
 	}
 
-	if (!can_load_obs_plugin) {
+	/*if (!can_load_obs_plugin) {
 		blog(LOG_WARNING,
 		     "Skipping module '%s' due to possible "
 		     "import conflicts",
 		     info->bin_path);
 		goto load_failure;
-	}
+	}*/
 
 	int code = obs_open_module(&module, info->bin_path, info->data_path);
 	switch (code) {
